docs/components/rich-text-editor.md

```diff
@@ -86,6 +86,7 @@ function MyEditor() {
 - `pasteFn` (optional): A function that will be called whenever the user pastes to the editor. The function will be called with a string of text and
     returns a string that will be pasted. If you wish to prevent a paste entirely, return `false` from the pasteFn. Note that pasted HTMl is vulnerable
     to cross site scripting. See [pasting docs](/concepts/pasting.md) for more details.
+- `sanitizeHTML` (optional but HIGHLY RECOMMENDED): A function that will be called with an HTML string as a parameter just prior to that HTML being inserted into the DOM. This happens whether the HTML was provided in initialHTML or is a result of a paste event. Remember, Bandicoot relies on the consumer to prevent XSS.
```

Comment:
Perhaps we should just mandate the prop?

Reply:
I think what you have here is good -- not strictly necessary but highly recommended

Reply:
I guess it is still TBD (pending the bandicoot issue PR approval), but we also should mention that it passes back a string that represents what kind of action bandicoot is taking. i.e. ('setHTML', 'getHTML', etc.).  In other words it will pass (htmlString, actionTypeString) into the sanitizeHTML function.